tox.ini

```diff
@@ -74,6 +74,7 @@ deps =
     esmpy: git+https://github.com/esmf-org/esmf.git@v{env:ESMF_VERSION}\#subdirectory=src/addon/esmpy
     upstream: -r {TOXINIDIR}/CI/requirements_upstream.txt
     xclim: xclim=={env:XCLIM_VERSION}
+    xsdba: git+https://github.com/Ouranosinc/xsdba.git@main
```

Comment:
Is this temporary until xsdba 4 ? if so, maybe put a fixme, so we don't forget  to switch it back ?

Reply:
We need it for right now, but yes, it should be removed when v0.4.0 comes out. Good call.

I was thinking of keeping it to be used as an upstream test, but I just remembered we have `CI/requirements_upstream.txt`. I'll add it there.